sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizePiiEntitiesTests.cs

```diff
@@ -0,0 +1,169 @@
+            Assert.IsTrue(results[1].HasError);
+            InvalidOperationException ex = Assert.Throws<InvalidOperationException>(() => results[1].Entities.GetType());
+            Assert.AreEqual(exceptionMessage, ex.Message);
+        }
+
+        [Test]
+        public async Task RecognizePiiEntitiesBatchConvenienceTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            var documents = batchConvenienceDocuments;
+
+            RecognizePiiEntitiesResultCollection results = await client.RecognizePiiEntitiesBatchAsync(documents);
+
+            foreach (RecognizePiiEntitiesResult result in results)
+            {
+                Assert.GreaterOrEqual(result.Entities.Count(), 2);
+            }
+        }
+
+        [Test]
+        public async Task RecognizePiiEntitiesBatchConvenienceWithStatisticsTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            var documents = batchConvenienceDocuments;
+
+            RecognizePiiEntitiesResultCollection results = await client.RecognizePiiEntitiesBatchAsync(documents, "en", new RecognizePiiEntitiesOptions { IncludeStatistics = true });
+
+            foreach (RecognizePiiEntitiesResult result in results)
+            {
+                Assert.GreaterOrEqual(result.Entities.Count(), 2);
+            }
+
+            Assert.IsNotNull(results.Statistics.DocumentCount);
+            Assert.IsNotNull(results.Statistics.InvalidDocumentCount);
+            Assert.IsNotNull(results.Statistics.TransactionCount);
+            Assert.IsNotNull(results.Statistics.ValidDocumentCount);
+        }
+
+        [Test]
+        public async Task RecognizePiiEntitiesBatchTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            List<TextDocumentInput> documents = batchDocuments;
+
+            RecognizePiiEntitiesResultCollection results = await client.RecognizePiiEntitiesBatchAsync(documents);
+
+            foreach (RecognizePiiEntitiesResult result in results)
+            {
+                Assert.GreaterOrEqual(result.Entities.Count(), 2);
+            }
+        }
+
+        [Test]
+        public async Task RecognizePiiEntitiesBatchWithStatisticsTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            List<TextDocumentInput> documents = batchDocuments;
+
+            RecognizePiiEntitiesResultCollection results = await client.RecognizePiiEntitiesBatchAsync(documents, new RecognizePiiEntitiesOptions { IncludeStatistics = true });
+
+            foreach (RecognizePiiEntitiesResult result in results)
+            {
+                Assert.GreaterOrEqual(result.Entities.Count(), 2);
+            }
+
+            Assert.IsNotNull(results.Statistics.DocumentCount);
+            Assert.IsNotNull(results.Statistics.InvalidDocumentCount);
+            Assert.IsNotNull(results.Statistics.TransactionCount);
+            Assert.IsNotNull(results.Statistics.ValidDocumentCount);
```

Comment:
Same here.

Reply:
Thanks! now that we have codegen, these checks don't make sense.
I updated it and created issue https://github.com/Azure/azure-sdk-for-net/issues/14833 to clean the tests of other endpoints